cypress.json

```diff
@@ -0,0 +1,4 @@
+{
+  "baseUrl": "http://localhost:8000/",
+  "ignoreTestFiles": ["**/*.skip.*", "/**/*.skip/*"]
```

Comment:
I decided to leave the example Cypress specs in here to hopefully help out anybody working on the build who's not familiar with Cypress. Happy to trash them all and point to their docs in the readme if that makes more sense.